tutor/src/components/navbar/support-menu.jsx

```diff
@@ -122,7 +122,9 @@ class SupportMenuDropDown extends React.PureComponent {
         >
           <span>Help Articles</span>
         </MenuItem>
-        <SupportDocument courseId={courseId} />
+        <TourAnchor id="menu-support-document">
+          <SupportDocument courseId={courseId} />
+        </TourAnchor>
         <MenuItem
```

Comment:
The snapshots suggest that we've got an incorrect DOM here,  it's nesting as:
```html
<ul>
    <div>
          <li>
```

I think if we add the  `tagName="li" prop to `TourAnchor` it should fix this, but then `SupportDocument` will need updated to not have a `li` tag.

```jsx

<ul>
   <TourAnchor tagname="li" id="menu-support-document" className="support-document-link">
       <a href="https://d3bxy9euw4e147.cloudfr/openstax_tutor_student_getting_s" ... />
   </TourAnchor>
</ul>

```

Reply:
Oh thanks for catching and fixing!